src/bin/elasticsearch_exporter.rs

```diff
@@ -116,6 +116,11 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         exporter_metrics_lifetime_interval: opts.exporter_metrics_lifetime_interval.0.clone(),
         exporter_metrics_lifetime_default_interval: *opts
             .exporter_metrics_lifetime_default_interval,
+
+        elasticsearch_certificate_path: opts.elasticsearch_certificate_path.clone(),
+        elasticsearch_certificate_validation: opts
+            .elasticsearch_certificate_validation
+            .map(|opt| opt.0),
```

Comment:
Not very elegant :/

Reply:
Any reason you cannot use `CertificateValidationOptions`? I see this is needed to convert from a string, no?

Reply:
You have full `CertificateValidationOptions` ownership, this is an enum you own, I don't see a reason to not implement `FromStr` for it directly.

Reply:
Yeah let's do that, updated.